### PR TITLE
(FACT-1438) Fix tests on s390x

### PR DESCRIPTION
**Pull request by johnduarte**
Fix `acceptance/tests/facts/el.rb` to accommodate s390x instances.

[skip-ci]

### CODE REVIEW

**Comment by johnduarte** [comment]
This change has been successfully run against the following platforms:

- redhat-6-s390x
- centos-6-i386
- scientific-6-i386
- oracle-6-i386

File: acceptance/tests/facts/el.rb [hunk 1]
```diff
@@ -10,6 +10,8 @@
 # Don't test on RedHat 4, as is does not include yum (See BKR-512)
 confine :to, :platform => /el-[5-7]|centos-[4-7]/
 
+@ip_regex = /^([1-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])(\.([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])){3}$/
```
Comment (MikaelSmith) [inline acceptance/tests/facts/el.rb]: Why not go with something like https://www.safaribooksonline.com/library/view/regular-expressions-cookbook/9780596802837/ch07s16.html? Or are you specifically disallowing `0.0.0.0` (as an example).

Comment (johnduarte) [inline acceptance/tests/facts/el.rb]: Thanks for the suggestion @MikaelSmith . I have updated the regex in the el.rb file and the previously committed sles.rb file.

**Comment by MikaelSmith** [comment]
Works for me.

**Pull request merged**